app/presenters/summary/sections/case_details.rb

```diff
@@ -44,6 +55,10 @@ def answers
       def kase
         @kase ||= crime_application.case
       end
+
+      def case_concluded?
+        kase.has_case_concluded == 'yes' && !kase.date_case_concluded.nil?
```

Comment:
If this is a ValueObject I think we can just use `has_case_concluded.yes?` rather than doing the direct comparison with the string.